components/hab/src/cli.rs

```diff
@@ -942,23 +940,23 @@ pub fn sub_sup_run() -> App<'static, 'static> {
         "Application name; [default: not set].")
     (@arg ENVIRONMENT: --environment -e +takes_value requires[APPLICATION]
         "Environment name; [default: not set].")
-    (@arg GROUP: --group +takes_value
-        "The service group; shared config and topology [default: default].")
+    (@arg GROUP: --group default_value("default")
+        "The service group; shared config and topology.")
     (@arg TOPOLOGY: --topology -t +takes_value possible_value[standalone leader]
         "Service topology; [default: none]")
-    (@arg STRATEGY: --strategy -s +takes_value {valid_update_strategy}
-        "The update strategy; [default: none] [values: none, at-once, rolling]")
+    (@arg STRATEGY: --strategy -s default_value("none") {valid_update_strategy}
```

Comment:
I think we can get rid of the explicit values listing on the next line if we do it this way:
```suggestion
    (@arg STRATEGY: --strategy -s default_value("none") {valid_update_strategy} possible_value("at-once") possible_value("rolling")
```
I'm not sure if that means we should eliminate `valid_update_strategy` or not. It's just a question of more redundant code versus having an error caught a bit earlier.

Similarly for `BINDING_MODE`.

Reply:
I thought about that, but the way the validation function was working made me pause. My thinking was that switching to the builder style pattern for the cli would make it easier to leverage `protocol::types::UpdateStrategy`. The more I look at it though, the more that I think that you may be right.